dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/RequestParamsValidator.java

```diff
@@ -496,16 +596,33 @@ public static void validateUnsupportedParameter(
 
   private static QueryFilter operatorValueQueryFilter(String operator, String value, String filter)
       throws BadRequestException {
-    if (StringUtils.isEmpty(operator) || StringUtils.isEmpty(value)) {
+    if (StringUtils.isEmpty(operator)) {
       throw new BadRequestException(INVALID_FILTER + filter);
     }
 
+    QueryOperator queryOperator;
     try {
-      return new QueryFilter(QueryOperator.fromString(operator), escapedFilterValue(value));
-
+      queryOperator = QueryOperator.fromString(operator);
     } catch (IllegalArgumentException exception) {
       throw new BadRequestException(INVALID_FILTER + filter);
     }
+
+    if (queryOperator == null) {
+      throw new BadRequestException(INVALID_FILTER + filter);
+    }
+
+    if (queryOperator.isUnary()) {
```

Comment:
Please tell the user which operator is used incorrectly. Always think about how you would fix an issue if you were the user given the info that you get here.